### PR TITLE
Convert panels implemented

### PR DESCRIPTION
**Pull request by MelissaRoman06**
View classes organized in packages. Class names changed to more descriptive ones

### CODE REVIEW

File: src/main/java/com/fundation/logic/controller/Controller.java [hunk 1]
```diff
@@ -277,12 +277,12 @@ public void setEvents() {
     }
 
     public void showLoadSaveData(){
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/controller/Controller.java]: put space before "{"

File: src/main/java/com/fundation/logic/model/CriteriaRecord.java [hunk 2]
```diff
@@ -77,4 +77,12 @@ public String getType() {
     public String getJson() {
         return this.json;
     }
+
+    public boolean equals(CriteriaRecord record) {
+        if(this.id == record.id && this.date.equals(record.date) && this.name.equals(record.name) &&
+                this.type.equals(record.type) && this.json.equals(record.json)){
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/model/CriteriaRecord.java]: put space before "{"

File: src/main/java/com/fundation/logic/view/MainFrame.java [hunk 3]
```diff
@@ -9,33 +9,36 @@
  */
 package com.fundation.logic.view;
 
-import javax.swing.JFrame;
-import javax.swing.JPanel;
-import javax.swing.JScrollPane;
-import javax.swing.JSplitPane;
+import com.fundation.logic.view.resultTable.TableResult;
+
+import javax.swing.*;
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/MainFrame.java]: Import only the needed classes, not use *

File: src/main/java/com/fundation/logic/view/MainFrame.java [hunk 4]
```diff
@@ -9,33 +9,36 @@
...
 import javax.swing.border.EmptyBorder;
-import java.awt.BorderLayout;
+import java.awt.*;
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/MainFrame.java]: Import only the needed classes, not use *

File: src/main/java/com/fundation/logic/view/MainTabs.java [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+ */
+package com.fundation.logic.view;
+
+import com.fundation.logic.view.loadSaveCriteria.LoadSavePanel;
+import com.fundation.logic.view.convert.SplitPanelConvert;
+import com.fundation.logic.view.loadSaveCriteria.SplitPanelDate;
+import com.fundation.logic.view.search.GeneralSearchPanel;
+import com.fundation.logic.view.search.SearchSplitPanel;
+
+import javax.swing.*;
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/MainTabs.java]: Import only the needed classes, not use *

File: src/main/java/com/fundation/logic/controller/Controller.java (hunk 1 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/controller/Controller.java]: done

File: src/main/java/com/fundation/logic/model/CriteriaRecord.java (hunk 2 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/model/CriteriaRecord.java]: done

File: src/main/java/com/fundation/logic/view/MainTabs.java (hunk 5 above)
Comment (MelissaRoman06) [reply src/main/java/com/fundation/logic/view/MainTabs.java]: done

File: src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java [hunk 6]
```diff
@@ -0,0 +1,364 @@
...
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.view.convert;
+
+import com.fundation.logic.view.PanelSetter;
+
+import javax.swing.*;
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java]: Import only the needed classes, not use *

File: src/main/java/com/fundation/logic/view/convert/SplitPanelConvert.java [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.view.convert;
+
+import javax.swing.*;
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/convert/SplitPanelConvert.java]: Import only the needed classes, not use *

File: src/main/java/com/fundation/logic/view/search/BasicSearchPanel.java [hunk 8]
```diff
@@ -7,78 +7,82 @@
...
+import com.fundation.logic.view.PanelSetter;
 
 import javax.swing.JPanel;
 import javax.swing.JTextField;
 import javax.swing.JLabel;
 import javax.swing.JFileChooser;
 import javax.swing.JButton;
-import java.awt.GridBagConstraints;
-import java.awt.Font;
+import java.awt.*;
```
Comment (aburgosu) [inline src/main/java/com/fundation/logic/view/search/BasicSearchPanel.java]: Import only the needed classes, not use *

**Pull request merged**